modules/net-lb-app-ext/main.tf

```diff
@@ -28,6 +28,12 @@ locals {
     [for k, v in google_compute_ssl_certificate.default : v.id],
     [for k, v in google_compute_managed_ssl_certificate.default : v.id]
   )
+  trust_config_name = "trust-config-${var.name}"
```

Comment:
we don't prefix names with the resource in modules, usually; the name itself is fine